src/TLinkStore.cpp

```diff
@@ -50,6 +50,16 @@ QStringList& TLinkStore::getHints(int id)
     return mHintStore[id];
 }
 
+QStringList TLinkStore::getLinksConst(int id) const
+{
+    return mLinkStore[id];
+}
+
+QStringList TLinkStore::getHintsConst(int id) const
+{
+    return mHintStore[id];
```

Comment:
Same as previous method.

Reply:
I changed it also for getCurrentLinks above as it is also const